tests/test_service_routes.py

```diff
@@ -385,6 +385,31 @@ def test_update_shopcart_item(self):
         self.assertEqual(updated_shopcart_item["sku"], 1001)
         self.assertEqual(updated_shopcart_item["name"], "item_1")
         self.assertEqual(updated_shopcart_item["amount"], 4)
+    
+    def test_update_shopcart_item_with_non_existing_shopcart(self):
+        """ Update an item where shopcart doesn't exists """
+
+        # create a shopcart item to update
+        new_shopcart_item = { "price":50.00}
+        resp = self.app.put(
+            "/shopcartitems/{}".format(1),
+            json=new_shopcart_item,
+            content_type="application/json",
+        )
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_query_shopcart_item(self):
+        """ Query shopcart item list without any query string """
+        test_shopcart = ShopcartFactory()
+        resp = self.app.post("/shopcarts", json={"user_id": test_shopcart.user_id}, content_type="application/json")
```

Comment:
I think this line will give an error in pylint for being too long

Reply:
I didn't realize we need to run pylint for test files. Should I do a bulk update to fix other syntax issues in tests?

Reply:
oh I thought it was for all the .py files. the score is above 9.0 for them right now